srcpkgs/devilution/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'devilutionx'
+pkgname=devilutionx
+version=1.0.0
+revision=1
+wrksrc="devilutionX-${version}"
+build_style=cmake
+configure_args="-DBINARY_RELEASE=ON -DTTF_FONT_PATH=\"/usr/share/fonts/truetype/CharisSILB.ttf\""
+makedepends="SDL2-devel SDL2_ttf-devel SDL2_mixer-devel libsodium-devel"
+depends="SDL2_mixer SDL2_ttf libsodium"
```

Comment:
Line useless.

Reply:
One line of dependencies for compilation, another line as package dependencies, are useful lines.

Reply:
Depends should not be listed. These dependencies will be automatically determined.

Reply:
Removed the depends line.